application/checks/check_bsdd_v2.py

```diff
@@ -145,9 +143,13 @@ def check_bsdd(ifc_fn, task_id):
                                 bsdd_result.classification_code = bsdd_content["code"]
                                 bsdd_result.classification_domain = domain_name
 
-                                bsdd_result.val_ifc_type = sum(ifc_instance.is_a(t) for t in bsdd_content["relatedIfcEntityNames"]) >= 1
+                                if "relatedIfcEntityNames" in bsdd_content.keys():
+                                    bsdd_result.val_ifc_type = sum(ifc_instance.is_a(t) for t in bsdd_content["relatedIfcEntityNames"]) >= 1
```

Comment:
`sum()` is not wrong, but what you're basically writing is:

`any(ifc_instance.is_a(t) for t in bsdd_content["relatedIfcEntityNames"])`

which is a bit better understandable